packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -1,5 +1,22 @@
 {
     "types": [
+        {
+            "name": "languageServerSetupStage",
```

Comment:
I wish we could do properly namespaced things in telemetry. It sucks that we always have prefix specific types like this

instead of having:
```
namespace lsp {
    
}
```